Framework/DataHandling/src/LoadILLIndirect.cpp

```diff
@@ -149,7 +150,7 @@ void LoadILLIndirect::loadDataDetails(NeXus::NXEntry &entry) {
 
   m_numberOfTubes = static_cast<size_t>(data.dim0());
   m_numberOfPixelsPerTube = static_cast<size_t>(data.dim1());
-  m_numberOfChannels = static_cast<size_t>(data.dim2());
+  m_numberOfChannels = static_cast<size_t>(data.dim2()) + 1; // +1 For bin edges
```

Comment:
Whenever `m_numberOfChannels` was used throughout the code one was always being appended to it so I moved it to assignment instead